stash/test/prices.test.ts

```diff
@@ -72,15 +72,15 @@ describe.skip('test pricing processor', () => {
     //asset_10 (1000 pools)
     store.get(1)!.length.should.be.equal(fixtures.LEN)
     // asset_11(500 pools) + asset_18 dependent of 7 (750 prices)
-    store.get(2)!.length.should.be.equal((fixtures.LEN / 4) * 5)
```

Comment:
We get different data than before, tests match what we get but QA team to take a look at this in the future

Reply:
AFIK those tests were created by sm1 else, not QA  . and No clue about those magic numbers :S